manifests/slave.pp

```diff
@@ -150,7 +150,7 @@
     ensure => 'file',
     mode   => '0755',
     owner  => 'root',
-    group  => 'root',
+    group  => '0',
```

Comment:
I've seen problems with the future parser, when the UID or GID is given literally and interpreted as integer, i.e.:
group => 0,

that usually caused trouble to me, since I run the future parser since a long time. However, '0' was the "safe" way to fix it ;)